resources/api.go

```diff
@@ -110,6 +114,8 @@ type Api interface {
 	Post(path string, handler faas.HttpMiddleware, opts ...MethodOption)
 	Delete(path string, handler faas.HttpMiddleware, opts ...MethodOption)
 	Options(path string, handler faas.HttpMiddleware, opts ...MethodOption)
+	// compatible with http.HandleFunc
+	HandleFunc(pattern string, handler func(http.ResponseWriter, *http.Request), opts ...MethodOption)
```

Comment:
I would just call this `All`

Reply:
I was trying to highlight that this is like the http server HandleFunc(). Not that it registers all methods